Apps/UnitTests/Shared/Tests.h

```diff
@@ -12,25 +13,17 @@
 #include <thread>
 #include <napi/env.h>
 
-std::promise<int32_t> exitCode;
+Babylon::Graphics::Configuration deviceTestConfig{};
 
-static inline constexpr const char* JS_FUNCTION_NAME{ "SetExitCode" };
-void SetExitCode(const Napi::CallbackInfo& info)
+TEST(JSTest, JavaScriptTests)
 {
-    Napi::Env env = info.Env();
-    if (info.Length() < 1)
-    {
-        throw Napi::TypeError::New(env, "Must provide an exit code");
-    }
-    exitCode.set_value(info[0].As<Napi::Number>().Int32Value());
-}
+    std::promise<int32_t> exitCode;
+    Babylon::Graphics::Device device = deviceTestConfig;
```

Comment:
This looks super strange.
```suggestion
    Babylon::Graphics::Device device{deviceTestConfig};
```